pkg/config/admin_server_config.go

```diff
@@ -120,3 +120,7 @@ func (c *AdminAPIServerConfig) ObservabilityExporterConfig() *observability.Conf
 func (c *AdminAPIServerConfig) IsMaintenanceMode() bool {
 	return c.MaintenanceMode
 }
+
+func (c *AdminAPIServerConfig) GetAuthenticatedSMSFailClosed() bool {
```

Comment:
just `AuthenticatedSMSFailClosed` ?

Reply:
I was trying to match the existing things in the interface

Reply:
yeah - that's fine.